audit-ci.json

```diff
@@ -3,12 +3,6 @@
   "pass-enoaudit": true,
   "retry-count": 20,
   "allowlist": [
-    {
```

Comment:
Why was this removed?

Reply:
It was expired, so didn't seem like it was needed anymore.